jwst/extract_1d/extract.py

```diff
@@ -2753,61 +2777,77 @@ def do_extract1d(input_model, ref_dict, smoothing_length=None,
-                            "so apply_nod_offset will be set to False",
-                            source_type)
+                log.warning(
+                    f"SRCTYPE = {source_type}'; correcting for nod/dither offset will only be done for a point source, "
+                    f"so apply_nod_offset will be set to False"
+                )
             extract_params = get_extract_parameters(
-                                ref_dict,
-                                slit, slit.name, sp_order,
-                                input_model.meta, smoothing_length, bkg_order,
-                                apply_nod_offset)
+                extract_ref_dict,
+                slit,
+                slit.name,
+                sp_order,
+                input_model.meta,
+                smoothing_length,
+                bkg_order,
+                apply_nod_offset
+            )
+
             if subtract_background is not None:
                 extract_params['subtract_background'] = subtract_background
+
             if extract_params['match'] == NO_MATCH:
                 log.critical('Missing extraction parameters.')
                 raise ValueError('Missing extraction parameters.')
             elif extract_params['match'] == PARTIAL:
-                log.info('Spectral order %d not found, skipping ...', sp_order)
+                log.info(f'Spectral order {sp_order} not found, skipping ...')
                 continue
-            extract_params['dispaxis'] = \
-                        slit.meta.wcsinfo.dispersion_direction
+
+            extract_params['dispaxis'] = slit.meta.wcsinfo.dispersion_direction
+
             if extract_params['dispaxis'] is None:
-                log.warning("The dispersion direction information is "
-                            "missing, so skipping ...")
+                log.warning("The dispersion direction information is missing, so skipping ...")
                 continue
+
             if photom_has_been_run:
                 pixel_solid_angle = slit.meta.photometry.pixelarea_steradians
+
                 if pixel_solid_angle is None:
                     pixel_solid_angle = 1.
-                    log.warning("Pixel area (solid angle) is not populated; "
-                                "the flux will not be correct.")
+                    log.warning("Pixel area (solid angle) is not populated; the flux will not be correct.")
             else:
                 pixel_solid_angle = 1.                  # not needed
 
             try:
-                (ra, dec, wavelength, temp_flux, background,
-                 npixels, dq, prev_offset) = extract_one_slit(
-                                        input_model, slit, -1,
-                                        prev_offset, True, extract_params)
+                ra, dec, wavelength, temp_flux, background, npixels, dq, prev_offset = extract_one_slit(
+                    input_model,
+                    slit,
+                    -1,
```

Comment:
Is there some way to make it more obvious exactly what this value is referring to in the argument list, e.g. "sp_order = -1" or whatever it is? Otherwise the reader/maintainer has to go find the extract_one_slit() function and count down through the list of arguments to figure out what it is.